### DISCUSSION TITLE
When inserting an image, the img tag does not conform to the MDX specification

### DISCUSSION THREAD

**Discussion by witcan** [GitbookIO/community#784] [0. General bugs and questions]
When inserting an image, the img tag has no ending symbol
![QQ_1730779382701](https://github.com/user-attachments/assets/026b5095-6ca3-472b-b7c6-9620761240de)

error:
Cause: Unexpected closing tag `</figure>`, expected corresponding closing tag for `<img>` (20:9-20:54)

**Comment by addisonschultz** [comment]
@witcan where are you seeing this - is this what’s generated through GitSync?
